schemas/mozilla_nimbus_schemas/tests/jetstream/test_analysis_errors.py

```diff
@@ -0,0 +1,212 @@
+import json
+
+import pytest
+from pydantic import ValidationError
+
+from mozilla_nimbus_schemas.jetstream import (
+    AnalysisBasis,
+    AnalysisError,
+    AnalysisErrors,
+)
+
+
+class TestAnalysisErrors:
```

Comment:
You're not subclassing `unittest.TestCase` so these don't need to be in a class.